### PR TITLE
chore: add alias doc

### PR DESCRIPTION
**Pull request by sanyuan0704**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: website/builder/src/en/guide/basic/alias.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Alias
```
Comment (chenjiahan) [inline website/builder/src/en/guide/basic/alias.md:1]: 文档链接好像没加到 vuepress config 里

File: website/builder/src/zh/guide/basic/alias.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# 路径别名
+
+在 Builder 中，你有两种方式可以设置路径别名:
+
+- 通过 `source.alias` 配置
```
Comment (chenjiahan) [inline website/builder/src/zh/guide/basic/alias.md:5]: source.alias 改成链接哈

File: website/builder/src/en/guide/basic/alias.md (hunk 1 above)
Comment (sanyuan0704) [reply website/builder/src/en/guide/basic/alias.md:1]: 可以，我补充下

File: website/builder/src/zh/guide/basic/alias.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  source: {
+    alias: (alias) => {
+      alias['@common'] = './src/common';
+      return alias;
+    },
+  },
+};
+```
+
+## 通过 `tsconfig.json` 中的 `paths` 配置
```
Comment (chenjiahan) [inline website/builder/src/zh/guide/basic/alias.md:39]: 可以补充说明一下，这是我们在 TS 项目中推荐使用的方式，因为可以同时解决类型问题

**Pull request merged**